src/main/java/chess/domain/board/Board.java

```diff
@@ -0,0 +1,86 @@
+package chess.domain.board;
+
+import chess.domain.piece.Empty;
+import chess.domain.piece.Piece;
+import chess.domain.piece.PieceColor;
+import chess.domain.position.Position;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class Board {
+
+    private static final Piece EMPTY_PIECE = new Empty();
+    private final Map<Piece, Position> coordinates;
+
+    public Board(Map<Piece, Position> coordinates) {
+        this.coordinates = coordinates;
+    }
+
+    public void putPiece(Piece piece, Position position) {
+        coordinates.put(piece, position);
+    }
+
+    public Board() {
+        this.coordinates = new LinkedHashMap<>();
+    }
```

Comment:
```suggestion
    public Board() {
        this(new LinkedHashMap<>());
    }
    
    public void putPiece(Piece piece, Position position) {
        coordinates.put(piece, position);
    }
```

생성자를 위로 올려주시면 가독성이 상승합니다 :) 

```
class A {
    상수(static final) 또는 클래스 변수
    인스턴스 변수
    생성자
    팩토리 메소드
    메소드
    기본 메소드 (equals, hashCode, toString)
}
```

또한 인스턴스 변수의 초기화를 하나의 생성자에서 해주면 어떠한 장점이 있을지 고민해보아요